### ISSUE TITLE
ENH: Add "preflight" checklist to README

### ISSUE THREAD

**Issue by thermokarst** [qiime2/workshop-playbooks#18]
After [some](https://github.com/qiime2/workshop-playbooks/issues/16) of the [issues](https://github.com/qiime2/workshop-playbooks/issues/17) that cropped up in today's workshop, it seems like a checklist is in order for ensuring that things are good to go when spinning up a new workshop cluster. Off the top of my head:

- The necessary ports should be open and accessible (we haven't finished defining what that list is just yet)
- User account names are sane (we have had problems with odd names being generated in the past)
- A complete roster of attendees needs to be available, because until #11, you are stuck with what you ask for at provisioning time

@gregcaporaso, @antgonza, @ebolyen: you all recently dealt with spinning up a workshop cluster --- any thoughts on what else should be on a preflight checklist? Thanks!!

**Comment by antgonza** [comment]
Right, I think that list makes sense. Perhaps one more thing is Internet speed and a list of any restrictions in the network or anything we should be aware of. 

Now, I think it will be great if we could have one page (perhaps in github pages) that people can open and get some stats and verify the restrictions in their institution/building - at least now we know where we can test that it actually reports that 22 is blocked. 😄
